### PR TITLE
fix(options): negation using --no prefix

### LINKED ISSUE #194: --no-error still outputs errors
This works for me as an npm script:

```
    "efr": "eslint-find-rules --unused ~/.eslintrc -n"
```

This does not work. It outputs errors despite the --no-error flag.

```
    "efr": "eslint-find-rules --unused ~/.eslintrc --noerror"
```

I don't understand why it outputs errors anyway though.

### PR DESCRIPTION
**Pull request by sarbbottam**
see #195 for the context.

fixes #194

### CODE REVIEW

**Comment by sarbbottam** [comment]
>`--no-core`

Will raise a follow up PR. :)

**Comment by ta2edchimp** [comment]
LGTM!

File: src/bin/find.js [hunk 1]
```diff
@@ -7,21 +7,25 @@ var options = {
   getPluginRules: ['plugin', 'p'],
   getAllAvailableRules: ['all-available', 'a'],
   getUnusedRules: ['unused', 'u'],
-  n: ['no-error'],
-  nc: ['no-core'],
+  n: [],
+  error: ['e', 'error'],
```
Comment (sarbbottam) [inline src/bin/find.js]: 'e' is redundant

**Comment by sarbbottam** [comment]
Something is not right. I don't even see this commit in the master branch.

**Pull request merged**